src/ImmersedBoundaries/immersed_reductions.jl

```diff
@@ -20,6 +20,11 @@ const IF = AbstractField{<:Any, <:Any, <:Any, <:ImmersedBoundaryGrid}
 @inline condition_operand(func::Function,         op::IF, cond,      mask) = ConditionalOperation(op; func, condition=NotImmersed(cond),     mask)
 @inline condition_operand(func::Function,         op::IF, ::Nothing, mask) = ConditionalOperation(op; func, condition=NotImmersed(truefunc), mask)
 @inline condition_operand(func::typeof(identity), op::IF, ::Nothing, mask) = ConditionalOperation(op; func, condition=NotImmersed(truefunc), mask)
+@inline function condition_operand(func::Function, op::IF, cond::AbstractArray, mask)
```

Comment:
```suggestion

@inline function condition_operand(func::Function, op::IF, cond::AbstractArray, mask)
```

Let's add a line to help readability.